<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gesDossier</class>
 <widget class="QDialog" name="gesDossier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dossier</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Res.qrc">
    <normaloff>:/icones/Icones/doss4.jpg</normaloff>:/icones/Icones/doss4.jpg</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="repEdit">
       <property name="placeholderText">
        <string>Repertoire</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="clientEdit">
       <property name="placeholderText">
        <string>Client</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Dossier</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableView" name="dossierTableView">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btNouveau">
            <property name="text">
             <string>Nouveau</string>
            </property>
            <property name="icon">
             <iconset resource="../Res.qrc">
              <normaloff>:/icones/Icones/add.ico</normaloff>:/icones/Icones/add.ico</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btModifier">
            <property name="text">
             <string>Modifier</string>
            </property>
            <property name="icon">
             <iconset resource="../Res.qrc">
              <normaloff>:/icones/Icones/05-Edit-icon.png</normaloff>:/icones/Icones/05-Edit-icon.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btDepense">
            <property name="text">
             <string>Ajouter Depense</string>
            </property>
            <property name="icon">
             <iconset resource="../Res.qrc">
              <normaloff>:/icones/Icones/imgres3.jpg</normaloff>:/icones/Icones/imgres3.jpg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btSupprimer">
            <property name="text">
             <string>Supprimer</string>
            </property>
            <property name="icon">
             <iconset resource="../Res.qrc">
              <normaloff>:/icones/Icones/14-Delete-icon.png</normaloff>:/icones/Icones/14-Delete-icon.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Liste des conteneurs</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QTableView" name="contTableView">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="btTransport">
              <property name="text">
               <string>Transporter</string>
              </property>
              <property name="icon">
               <iconset resource="../Res.qrc">
                <normaloff>:/icones/Icones/transport-camion-vehicule-jaune-icone-4890-128.png</normaloff>:/icones/Icones/transport-camion-vehicule-jaune-icone-4890-128.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Res.qrc"/>
 </resources>
 <connections/>
</ui>
